cas-server-webapp/src/main/webapp/WEB-INF/webflow/login/login-webflow.xml

```diff
@@ -199,21 +199,21 @@
         <transition on="InvalidLoginLocationException" to="casBadWorkstationView"/>
         <transition on="InvalidLoginTimeException" to="casBadHoursView"/>
         <!-- Built-in exceptions that are explicitly used by OSF -->
-        <transition on="FailedLoginException" to="generateLoginTicket"/>
         <transition on="AccountNotFoundException" to="generateLoginTicket"/>
         <transition on="AccountDisabledException" to="casAccountDisabledView"/>
+        <transition on="FailedLoginException" to="generateLoginTicket"/>
         <!-- Customized exceptions for OSF -->
-        <transition on="InvalidVerificationKeyException" to="casInvalidVerificationKeyView" />
-        <transition on="AccountNotConfirmedOsfLoginException" to="casAccountNotConfirmedOsfLoginView" />
-        <transition on="AccountNotConfirmedIdPLoginException" to="casAccountNotConfirmedIdPLoginView" />
-        <transition on="ShouldNotHappenException" to="casShouldNotHappenView" />
-        <transition on="RemoteUserFailedLoginException" to="casRemoteUserFailedLoginView" />
+        <transition on="AccountNotConfirmedOsfLoginException" to="casAccountNotConfirmedOsfLoginView"/>
+        <transition on="AccountNotConfirmedIdPLoginException" to="casAccountNotConfirmedIdPLoginView"/>
+        <transition on="InvalidUserStatusException" to="casInvalidUserStatusView"/>
+        <transition on="InvalidVerificationKeyException" to="casInvalidVerificationKeyView"/>
         <transition on="OneTimePasswordRequiredException" to="casOtpLoginView"/>
         <transition on="OneTimePasswordFailedLoginException" to="casOtpLoginView"/>
+        <transition on="RemoteUserFailedLoginException" to="casRemoteUserFailedLoginView"/>
         <!-- Customized exceptions for delegated login -->
-        <transition on="DelegatedLoginException" to="delegatedLoginGenericErrorView" />
         <transition on="CasClientLoginException" to="delegatedLoginCasErrorView"/>
         <transition on="OrcidClientLoginException" to="delegatedLoginOrcidErrorView"/>
+        <transition on="DelegatedLoginException" to="delegatedLoginGenericErrorView"/>
```

Comment:
Just a re-order (alphabetically) with one rename "shouldNotHappen" to "invalidUserStatus".